backend/core/fileupload/migrations/0001_initial.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.8 on 2021-11-15 08:44
```

Comment:
Please remove the `migrations/*` files from the repository as they only make sense for maintaining an existing db.